libs-scala/resources-akka/src/test/suite/scala/com/digitalasset/resources/akka/AkkaResourceOwnerSpec.scala

```diff
@@ -4,9 +4,8 @@
 package com.daml.resources.akka
 
 import java.util.concurrent.atomic.AtomicBoolean
-
```

Comment:
Please configure your editor to leave this newline alone.

Reply:
Weird, it was working fine previously.